nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Basic front end

Fixes:
-N/A

Changes proposed in this request & reasons behind organizational or architectural decisions:
-Basic front end is added with full functionality for "Employee" resource

Steps to test:
-Pull down branch
-navigate to BangazonAPIFrontEnd/bangazon-api-front-end
-run npm install
-run npm-start


System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A

Commit message:
-

Link to feature ticket:
-

@ambitions-albatrosses
